ETL/ETL/_bezier.h

```diff
@@ -591,6 +506,31 @@ class bezier : public bezier_base<V,T>
 	}
 
 private:
+	/** Cubic Bezier curve */
+	static constexpr unsigned int DEGREE = 3;
+	/** Degree of eqn to find roots of */
+	static constexpr unsigned int W_DEGREE = 5;
+	/** Take binary sign of a, either -1, or 1 if >= 0 */
+	template<typename U>
+	static int sgn(const U& a)
+	{
+		return a < 0 ? -1 : 1;
+	}
+
+	/** Find minimum of a and b */
+	template<typename U>
+	static U min(const U& a, const U& b)
+	{
+		return a < b ? a : b;
+	}
+
+	/** Find maximum of a and b */
+	template<typename U>
+	static U max(const U& a, const U& b)
+	{
+		return a > b ? a : b;
+	}
+
```

Comment:
@ice0 should I simply `#include <algorithm>` to use `std::min()` and `std::max()` or is it better like this?

Reply:
<algorithm> is about 400KB so I prefer the current solution